src/windows/SliderWindows/SliderView.cpp

```diff
@@ -97,7 +97,8 @@ namespace winrt::SliderWindows::implementation {
                     this->ClearValue(xaml::Controls::Slider::StepFrequencyProperty());
                 }
                 else {
-                    this->StepFrequency(propertyValue.AsDouble());
+                    auto&& step = CalculateStepFrequencyPercentageValue(propertyValue.AsDouble());
```

Comment:
The advantage of that approach (comparing to just making a constant default step) is that it provides the `<Slider/>` with the same resolution no matter what length is the `<Slider/>`.

Reply:
Good idea! Worth noting, though, that this assumes min/max values are specified, which they aren't always.